From 13978ffae8d3aacb5ddeabf08c47611d9079722e Mon Sep 17 00:00:00 2001
From: Hai Pham <hai.pham.ud@renesas.com>
Date: Tue, 27 May 2025 21:42:02 +0700
Subject: [PATCH 47/52] configs: ironhide: Downgrade SD/MMC from UHS/HS200

Downgrade the eMMC to HS/HS52 mode and/or SD card
to non-UHS mode in the initial support of SD/MMC.

Signed-off-by: Hai Pham <hai.pham.ud@renesas.com>
---
 configs/renesas_rcar5.config | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/configs/renesas_rcar5.config b/configs/renesas_rcar5.config
index 3aa387fc0ed..b82400cf8ba 100644
--- a/configs/renesas_rcar5.config
+++ b/configs/renesas_rcar5.config
@@ -6,9 +6,9 @@ CONFIG_ARCH_CPU_INIT=y
 CONFIG_BOOTARGS="root=/dev/nfs rw nfsroot=192.168.0.1:/export/rfs ip=192.168.0.20"
 CONFIG_CMD_MMC=y
 CONFIG_DM_ETH_PHY=y
-CONFIG_MMC_HS200_SUPPORT=y
-CONFIG_MMC_IO_VOLTAGE=y
-CONFIG_MMC_UHS_SUPPORT=y
+# CONFIG_MMC_HS200_SUPPORT is not set
+# CONFIG_MMC_IO_VOLTAGE is not set
+# CONFIG_MMC_UHS_SUPPORT is not set
 CONFIG_RENESAS_SDHI=y
 CONFIG_SPI_FLASH_SPANSION=y
 CONFIG_SYS_I2C_RCAR_I2C=y
-- 
2.34.1

